src/runtime/edbml/functions/ts.ui.tablemenu.edbml

```diff
@@ -22,10 +22,10 @@
 					if(table.menuopen) {
 						<th class="ts-table-choices">
 							<button data-action="select-page" class="ts-button ts-tertiary">
-								<span>This Page</span>
+								<span class="t">This Page</span>
```

Comment:
The translation in Tradeshift-ui is not working like in the V4. This is not working . Please create an issue, We have to plan and fix it.